src/coreclr/vm/peimage.cpp

```diff
@@ -887,7 +887,9 @@ void PEImage::EnumMemoryRegions(CLRDataEnumMemoryFlags flags)
 
 
 PEImage::PEImage():
+    m_path(),
```

Comment:
Are these additions really needed? Isn't the parameter-less constructor used by default?

Reply:
Only if you do not have a constructor, compiler may generate one. Once you have a constructor, compiler just calls it. 

`new PEImage()` would leave these uninitialized, but typically we were lucky except for OSX with optimizations (i.e. does not fail in Debug, but started failing in Checked/Release)